### DISCUSSION TITLE
Steam just open sourced their audio SDK. Could be alternative to BASS.

### DISCUSSION THREAD

**Discussion by Trung0246** [ppy/osu-framework#6192] [Ideas] [answered]
https://steamcommunity.com/games/596420/announcements/detail/7745698166044243233

Although I don't know feasibility of this to actually integrate into the codebase but sounds promising.

**Comment by smoogipoo** [comment, marked as answer]
This doesn't do anything for us. It has to do with spatial audio which isn't hugely relevant to 2D in general.
